src/helpers/scrollToId.js

```diff
@@ -0,0 +1,8 @@
+// Scrolls to the HTML element with the specified 'id' name
```

Comment:
I think that we shouldn't be worried about the element's ID. I think there is a way to functionally accomplish this same thing using element references instead.

Reply:
In this case (simply scrolling an element into view), compared to using the React `useRef` hook, utilizing the "id" attribute is simpler and results in a faster performance. Let me know your thought on this, thank you!